### DISCUSSION TITLE
[ 问题咨询 ] 提交sql工单怎样设置强制选择时间范围

### DISCUSSION THREAD

**Discussion by Ants5** [hhyo/Archery#1611] [Q&A]
### 问题描述
研发提交上线sql工单，没有选择可执行时间范围，在sql工单审批完成后执行会显示”不在可执行时间范围内，如果需要修改执行时间请重新提交工单“。确认主机系统时间是正常的

### 版本信息
 - 应用版本/分支：Release v1.8.5  
 - 部署方式：Docker-compose

**Comment by hhyo** [comment]
如果不选择执行时间范围，默认可以在任何时间段执行
